packages/cli/src/cli/autoformat.rs

```diff
@@ -97,7 +93,16 @@ async fn autoformat_project(check: bool) -> Result<()> {
     let crate_config = crate::CrateConfig::new(None)?;
 
     let mut files_to_format = vec![];
-    collect_rs_files(&crate_config.crate_dir, &mut files_to_format);
+
+    let gitignore_path = crate_config.crate_dir.join(".gitignore");
+    if gitignore_path.is_file() {
+        let gitigno = gitignore::File::new(gitignore_path.as_path()).unwrap();
+        if let Ok(files) = gitigno.included_files() {
+            files_to_format.extend(files)
```

Comment:
We should filter out any `.rs` files even if they are allowed in the gitignore. You might want to track a file in git that is not a rust file. Currently, I think this would try to format the `cargo.toml` file

Reply:
I felt like I was forgetting something but it completely slipped my mind, nice catch! gotta love reviews